index.js

```diff
@@ -1,29 +1,27 @@
 // ⭐️ Example Challenge START ⭐️
 
-/**
- * ### Challenge `processFirstItem`
+/**Example Task : processFirstItem()
+ * This example shows how you might go about solving the rest of the tasks
  * 
- * @instructions
- * Implement a higher-order function called `processFirstItem`.
- * It takes two arguments:
- * @param stringList an array of strings.
- * @param callback function that takes a string as its argument.
- * @returns the result of invoking `callback` with the FIRST element in `stringList`.
+ * Use the higher order function processFirstItem below to do the following:
+ *  1. Receive an array of strings called `stringList`
+ *  2. Receive a function that takes a string as its argument called `callback`
+ *  3. Return the result of invoking `callback` with the FIRST element in `stringList`
  * 
- * Example of usage of this higher-order function:
- * Invoking `processFirstItem` passing `['foo', 'bar']` and `(str) => str + str`,
- * should return 'foofoo'.
+ * For example: Invoking `processFirstItem` and passing `['foo', 'bar']` to stringList
```

Comment:
I may just remove this part - it kind of makes it more confusing? what do you think?

Reply:
let me reword it